src/constants/labels.json

```diff
@@ -138,7 +138,7 @@
     "header": {
       "overview": "Übersicht",
       "overviewExercises": "Übersicht der Übungen",
-      "cancelExercise": "Übung beenden",
+      "cancelExercise": "Zurück zur Übersicht",
```

Comment:
This also changes the wording of the button in other exercises, but not only in the vocabulary list. Also, as described in the ticket, the wording should only change to "Zurück zur Übersicht", if it is the last word in the list.

Reply:
ahh so. verstanden

Reply:
Please revert